src/components/BankLinks/renderCarousel.tsx

```diff
@@ -0,0 +1,58 @@
+import { JSX } from '@redneckz/uni-jsx';
+import { useState } from '@redneckz/uni-jsx/lib/hooks';
+import type { VNode } from '../../model/VNode';
+import { renderArrows } from '../../ui-kit/Button/renderArrows';
+
+interface CarouselProps {
+  slides: VNode[];
+  width: number;
+}
+
+interface RenderCarouselProps extends CarouselProps {
+  slidesToShow?: number;
+}
+
+interface CarouselWrapperProps extends CarouselProps {
+  activeIndex: number;
+}
+
+export const renderCarousel = JSX<RenderCarouselProps>(({ slidesToShow = 1, slides, width }) => {
```

Comment:
Это уже не redner-функция, это уже компонент.

```suggestion
export const Carousel = JSX<RenderCarouselProps>(({ slidesToShow = 1, slides, width }) => {
```